udp-relay.py

```diff
@@ -34,9 +34,13 @@ def fail(reason):
 sys.stderr.write('All set.\n')
 while True:
 	data, addr = s.recvfrom(32768)
-	if knownClient is None:
+	if knownClient is None or addr != knownServer:
+		print("")
```

Comment:
Remove this print? Looks like a debug leftover.

Reply:
The newline is there to visually separate into req-resp pairs:

```
All set.

Connection received from ('127.0.0.1', 33127)
	sending to ('192.168.56.104', 1812)
Connection received from ('192.168.56.104', 1812)
	sending to ('127.0.0.1', 33127)

Connection received from ('127.0.0.1', 56700)
	sending to ('192.168.56.104', 1812)
Connection received from ('192.168.56.104', 1812)
	sending to ('127.0.0.1', 56700)
```